packages/dnb-eufemia/src/fragments/drawer-list/style/dnb-drawer-list.scss

```
@@ -83,7 +83,7 @@
     }
 
     // higher than modal --modal-z-index:
-    z-index: 3200;
+    z-index: calc(var(--modal-z-index) + 200);
```

```suggestion
    z-index: calc(var(--modal-z-index, 3200) + 200);
```

We probably need to still have a fallback, in case the variable is not present.